services/tables/src/main/java/com/linkedin/openhouse/tables/common/DefaultReplicationInterval.java

```diff
@@ -0,0 +1,20 @@
+package com.linkedin.openhouse.tables.common;
+
+/**
+ * ENUM for default replication interval associated with Interval in {@link
+ * com.linkedin.openhouse.tables.api.spec.v0.request.components.ReplicationConfig}
+ */
+public enum DefaultReplicationInterval {
+  // default interval to run replication jobs
+  DAILY("24H");
```

Comment:
Do we plan to add any othe enum here? How do we represent interval like 48h or 72h? Where this enum is used?

Reply:
this enum is being used in the `PoliciesSpecMapper` to map empty interval parameters to a daily replication job schedule. I can adjust the naming of the class to be more clear. intervals like `48H` and `72H` would be provided by the user but `24H` is the default value if no interval value is provided